data/saas/config/sentry_config.yml

```diff
@@ -80,11 +80,11 @@ saas_config:
                 - dataset: sentry_connector
                   field: issues.id
                   direction: from
-          body: "{\"assignedTo\": \"\"}"
+          body: '{"assignedTo": ""}'
```

Comment:
Note the original body was intentional `body: "{\"assignedTo\": \"\"}"`, there's no test for this yet because we don't have data for this endpoint in our sentry instance. 

I believe when I tested against the sentry sandbox, specifying the body like this ` '{"assignedTo": ""}'` caused a bad request, how the body needs to be specified is pretty brittle currently, but I'll verify again in https://github.com/ethyca/fidesops/issues/343